enhancements/cert-manager/istio-csr-controller.md

```diff
@@ -0,0 +1,878 @@
+---
+title: istio-csr-controller
+authors:
+  - "@bhb"
+reviewers:
+  - "@tgeer" ## reviewer for cert-manager component
+approvers:
+  - "@tgeer" ## approver for cert-manager component
+api-approvers:
+  - "@tgeer" ## approver for cert-manager component
+creation-date: 2024-01-22
+last-updated: 2024-05-13
+tracking-link:
+  - https://issues.redhat.com/browse/CM-234
+see-also:
+  - NA
+replaces:
+  - NA
+superseded-by:
+  - NA
+---
+
+# Extend cert-manager-operator to manage istio-csr.
+
+## Summary
+
+This enhancement describes the proposal to extend cert-manager-operator to deploy and manage the `istio-csr`
+solution. `OpenShift Service Mesh` facilitates the security of both intra-cluster and external service
```

Comment:
An elaboration about the istio-csr is required. Would `istio-csr solution` mean a controller or set of controllers?

Reply:
A controller will be used manage the `istio-csr`.